lib/core/include/irods_server_properties.hpp

```diff
@@ -45,92 +50,158 @@ namespace irods
 
     extern const std::string SERVER_CONFIG_FILE;
 
+    using json   = nlohmann::json;
+    using logger = irods::experimental::log;
```

Comment:
I'd recommend moving these into the functions that use them or into a cpp file to avoid clashing with symbols in other translation units. It is much safer to use the fully qualified type in headers.

The logging library doesn't appear to be used.